cudamapper/src/index_host_copy.cuh

```diff
@@ -30,11 +30,11 @@ public:
     /// \param window_size the number of adjacent k-mers in a window, adjacent = shifted by one basepair
     /// \param cuda_stream D2H copy is done on this stream
     /// \return - pointer to claragenomics::cudamapper::IndexCache
-    explicit HostCache(const Index& index,
-                       const read_id_t first_read_id,
-                       const std::uint64_t kmer_size,
-                       const std::uint64_t window_size,
-                       const cudaStream_t cuda_stream);
+    explicit IndexHostCopy(const Index& index,
```

Comment:
this constructor does not need to be `explicit`.

Reply:
done